linodecli/arg_helpers.py

```diff
@@ -398,11 +398,24 @@ def action_help(cli, command, action):
                 if op.method in {"post", "put"} and arg.required
                 else ""
             )
-            nullable_fmt = " (nullable)" if arg.nullable else ""
-            print(
-                f"  --{arg.path}: {is_required}{arg.description}{nullable_fmt}"
+
+            extensions = []
+
+            if arg.format == "json":
+                extensions.append("JSON")
+
+            if arg.nullable:
+                extensions.append("nullable")
+
+            if arg.is_parent:
+                extensions.append("conflicts with children")
```

Comment:
I don't know if I like the wording here, any suggestions would be much appreciated!